test/specs/map/RealtimeLayersService.spec.js

```diff
@@ -195,8 +198,8 @@ describe('ga_realtimelayers_service', function() {
       var layer = map.getLayers().item(0);
       expect(layer.getSource().getFeatures().length).to.be(1);
 
-      // Here we don't test the change of the url because it's done in another
-      // service, we only test if the data are requetsed on language change.
+      var newLang = 'custom2';
+      dataUrl = dataUrl.replace('_custom', '_' + newLang);
```

Comment:
The meaning of the comment was : 'The RealtimeLayersService is not responsible of changing the url' so here you test something unrelated to the RealtimeLayersService . 

On language change the  RealtimeLayersService  reload  the geojsonUrl property it doesn't care of what's in it so the test was good no need to complexify more.